tox.ini

```
@@ -24,9 +24,10 @@ commands =
 [testenv:lint]
 deps =
     pre-commit
+    build
     twine
 commands =
     pre-commit run --all-files
-    python setup.py sdist
+    python -m build --sdist
```

Let's generate and check the wheel too:

```suggestion
    python -m build
```